test/remove_white_space.js

```diff
@@ -0,0 +1,3 @@
+const parser = require('../lib/index.js');
```

Comment:
Is this really needed? Can we merge this file in the main test file?

Reply:
<img width="1099" alt="스크린샷 2022-02-28 19 21 52" src="https://user-images.githubusercontent.com/49112423/155966390-ed12f7ee-c7d3-484f-964b-71aff49f2e09.png">

```txt
Before : \bjavascript://google.com/%0aalert(1)
After : javascript://google.com/%0aalert(1)

Before == After => Error occur
```
I tested like above. However, an error occurs because the url passed as an argument and the url after parsing are not the same. This means it worked fine. If there is no error in the above case, it means that the white space character was not removed after parsing.